web_src/less/_repository.less

```diff
@@ -1908,6 +1940,36 @@
         }
     }
 
+    .ui.attached.isSigned.isVerifiedUntrusted {
+        &:not(.positive) {
+            border-left: 1px solid #c2c193;
+            border-right: 1px solid #c2c193;
+        }
```

Comment:
Can `:not` selector be avoided? I think it's generally bad practice as it adds unexpected specificity to the rule. If possible, use a second rule to handle the `.positive` case.

Reply:
It's just a copy of the previous less.

Reply:
Maybe we can share the rules by moving it outside the parent selector?

Reply:
OK I've restructured. I'm gonna resolve so we can rediscuss based on new structure